### ISSUE TITLE
Estimate for training time

### ISSUE THREAD

**Issue by jlidard** [sshaoshuai/MTR#54]
Thank you for providing the code and for your helpful responses so far. I was wondering if you could provide an estimate for the (real-time, in hours/days) training time required for both MTR and MTR-e2e, for the training parameters mentioned in the paper.

I am training a similar model, but it is hard to debug with such long wait times. Any tips on this would be appreciated. Did you find that batch normalization accelerated convergence?

**Comment by justin-lidard-tri** [comment]
This is good info. Thanks for your reply!

**Comment by jlidard** [comment]
This is good info. Thanks for your reply!
